src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -791,17 +796,24 @@ protected function beginAlterByCopyTable($tableName)
     {
         $instructions = new AlterInstructions();
         $instructions->addPostStep(function ($state) use ($tableName) {
+            $tmpTableName = "tmp_{$tableName}";
             $createSQL = $this->getDeclaringSql($tableName);
 
-            $tmpTableName = 'tmp_' . $tableName;
-            $this->execute(
-                sprintf(
-                    'ALTER TABLE %s RENAME TO %s',
-                    $this->quoteTableName($tableName),
-                    $this->quoteTableName($tmpTableName)
-                )
+            // Table name in SQLite can be hilarious inside declaring SQL:
+            // - tableName
+            // - `tableName`
+            // - "tableName"
+            // - [this is a valid table name too!]
+            // - etc.
+            // Just remove all characters before first "(" and build them again
+            $createSQL = preg_replace(
+                "/^CREATE TABLE .* \(/Ui",
```

Comment:
```suggestion
                "/^CREATE TABLE [^(]+ \(/Ui",
```

- use `+` because `*` can match 0 times
- use `[^(]` to match any char except `(`

Previous expression would match `t ( id integer` as table name in `CREATE TABLE ​t ( id integer (5))`

Reply:
Oh, ` U (PCRE_UNGREEDY)` flag modifies this behavior...